SGVHAK_Rover/templates/chassis_config.html

```diff
@@ -26,49 +26,74 @@
 <script type="text/javascript" src="{{url_for('static', filename='chassis_config.js')}}"></script>
 <input type="hidden" id="request_wheel_status" value="{{url_for('request_wheel_status')}}" />
 <div class="container">
-  <h4 class="center_header">Chassis Configuraton</h4>
```

Comment:
Since these got moved, do we still use the `.center_header` style anymore? If not, please remove from style.css.

Reply:
Good point, I removed this class from the stylesheet and checked it's not lurking in other html files.